sdk/spring/azure-spring-boot-starter/CHANGELOG.md

```diff
@@ -40,7 +42,28 @@
 ### New Features
 - Upgrade to `Spring Boot` [2.4.3](https://github.com/spring-projects/spring-boot/releases/tag/v2.4.3).
 
+
 ## 3.2.0 (2021-03-03)
+### Breaking Changes
+- For the required scopes in auth code flow, use `Directory.Read.All` instead of `Directory.AccessAsUser.All`. ([#18901](https://github.com/Azure/azure-sdk-for-java/pull/18901))
```

Comment:
Since it's in azure-spring-boot-starter instead of aad starter, need we claim that the auth code flow is for aad only?